src/pages/projects.js

```diff
@@ -775,6 +775,34 @@ const ProjectDescriptions = () => (
         </div>
       </div>
     </div>
+
+    <div className="project-box">
+      <TitleWithAnchor headerClassName="project-major-title projects-common-title" headerTag="h3">Tetragon</TitleWithAnchor>
+      <div className="project-minor-title">eBPF-based Security Observability &amp; Runtime Enforcement</div>
```

Comment:
I'd recommend adding the word "Linux" somewhere, like ply and tracee do on this page, unless this project also works on other runtimes.  But https://github.com/cilium/tetragon specifically says Linux.

Reply:
Makes sense to me lets add Linux. For now at least its Linux only, although would be interesting to see if other runtimes would be possible at some point.  There is not much that is inherently specific to Linux arch in the configuration and export layers, but lets revisit later. The main blocker at BPF side would be function attach points with enforcement I'm not sure if/when other runtimes might support this.

Reply:
Yep, makes sense, thank you! I added Linux into the description:
"... thanks to smart Linux in-kernel filtering and aggregation logic built directly into the eBPF-based kernel-level collector."